docs/quickstarts/insights-tasks-pre-conversion/insights-tasks-pre-conversion.yml

```diff
@@ -0,0 +1,189 @@
+  description: |-
+    Generate a pre-conversion analysis report that summarizes potential problems and suggests recommended solutions to help you convert to RHEL. 
+  introduction: |-
+    [Red Hat Insights can be used only to run the pre-conversion analysis on CentOS Linux 7 systems.]{{admonition important}}   
+           
+    In this quick start, you will connect and run a pre-conversion analysis on CentOS Linux machines on the Red Hat Hybrid Cloud Console. The task generates a report that summarizes potential problems and suggests recommended solutions to help you convert to RHEL.
+    <br>
+    <br>
+    Additional information:
+
+    - **This feature is currently only available in Preview mode.**
+
+    - It is possible to convert systems from other Linux distributions. [View supported conversion paths.](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/converting_from_an_rpm-based_linux_distribution_to_rhel/con_supported-conversion-paths_converting-from-a-linux-distribution-to-rhel)
+
+    - Depending on your requirements, you can convert to RHEL using a number of supported methods. [View supported conversion methods.](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/converting_from_an_rpm-based_linux_distribution_to_rhel/conversion-methods_converting-from-a-linux-distribution-to-rhel)
+    
+    - [Explore additional conversion considerations](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/converting_from_an_rpm-based_linux_distribution_to_rhel/con_planning-a-rhel-conversion_converting-from-a-linux-distribution-to-rhel) when planning your RHEL conversion.
+  tasks:
+    - title: Verify CentOS Linux machines are connected
+      description: |-
+        [The only Red Hat Insights service you can use with registered CentOS Linux systems is the RHEL conversion. All other Insights services are available only after the conversion to RHEL.]{{admonition note}} 
+
+        Before running the pre-conversion analysis in Red Hat Insights, you must connect your CentOS machines to Insights<br>  
+        
+        **To verify CentOS Linux machines are connected:**
+
+        1. Connect CentOS Linux machines to Insights.
+          <details>
+            <summary><span style="color:#0066CC">&nbsp**Need to connect a CentOS Linux machine?**</span></summary>
+            - Install the Red Hat client tools:
+                - Download the Red Hat GPG key:<br>
+                `# curl -o /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release https://www.redhat.com/security/data/fd431d51.txt`{{copy}}<br>
```

Comment:
`sudo` is necessary here, I'd rather that then assume or instruct to run directly as root.

Reply:
Above comment added on L44 provides the necessary context, thanks!